.idea/.gitignore

```diff
@@ -0,0 +1,5 @@
+# Default ignored files
```

Comment:
Can you exclude all the .idea stuff from the git repo? Many devs wouldn't be using WebStorm for JavaScript, and this just becomes noise.

Reply:
Removed .idea folder